workspaces/bulk-import/.changeset/neat-news-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-bulk-import': minor
```

Comment:
I know I said we should use minor more, but this looks really like a bugfix and patch is fine in that case.

```suggestion
'@red-hat-developer-hub/backstage-plugin-bulk-import': patch
```